src/views/SensorProvisioningGuide.jsx

```diff
@@ -69,7 +69,7 @@ function SensorProvisioningGuide() {
             <div style={styles.stepText}>
               <ol>
                 <li style={styles.stepTitle}>
-                  Write the command <code>particle setup usb-done</code> to a new Particle Boron device.
+                  Write the command <code>particle usb setup-done</code> to a new Particle Boron device.
```

Comment:
I think the image needs to be updated too for this.

Also, can you please add a line in the CHANGELOG?

Reply:
Thanks!